Drop-In/src/main/java/com/braintreepayments/api/DropInEvent.java

```diff
@@ -0,0 +1,116 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class DropInEvent implements Parcelable {
+
+    private final DropInEventType type;
+    private final Bundle payload;
+
+    static DropInEvent createSendAnalytics(String eventName) {
```

Comment:
```suggestion
    static DropInEvent createSendAnalyticsEvent(String eventName) {
```
Nitpick for consistency